Digital Twin/systemmodels/SaniterRefill.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
Filename (and therefor the System Model name) seems to be missing the `iz` in `Sanitizer`